components/spec/Storyshots.spec.js

```diff
@@ -3,5 +3,6 @@ import { mount } from "enzyme";
 import "jest-styled-components";
 
 initStoryshots({
-  renderer: mount
+  renderer: mount,
+  storyNameRegex: /^((?!.*?SkipStoryshot).)*$/
```

Comment:
Apparently looking for a string within a story name is the only way to skip a storyshot for a particular story.. 

It needs to be skipped since it relies on using fetch for the purposes of the example.